MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+module(
+    name = "rules_proto",
+    compatibility_level = 1,
+    repo_name = "rules_proto",
```

Comment:
You don't need this because the default repo name is already the module name.

Reply:
Done.